### PR TITLE
Implement `CairoRunner::get_perm_range_check_limits()`

### PR DESCRIPTION
**Pull request by azteca1998**
# Implement `CairoRunner::get_perm_range_check_limits()`

## Description

Implementation of `CairoRunner::get_perm_range_check_limits()` and its dependencies.

## Checklist
- [ ] Linked to Github Issue
- [ ] Unit tests added
- [ ] Integration tests added.
- [x] This change requires new documentation.
  - [ ] Documentation has been added/updated.

### CODE REVIEW

**Comment by azteca1998** [comment]
[CairoRunner::get_perm_range_check_limits()](https://trello.com/c/8mEKHZQP/53-cairorunnergetpermrangechecklimits-esteve-getpermrangechecklimits-builtinrunnergetrangecheckusage)

**Comment by azteca1998** [comment]
[BuiltinRunner::get_ratio()](https://trello.com/c/7eNsI5Wf/66-builtinrunnergetratio)

File: src/vm/runners/cairo_runner.rs [hunk 1]
```diff
@@ -410,6 +411,35 @@ impl CairoRunner {
         Ok(())
     }
 
+    pub fn get_perm_range_check_limits(
```
Comment (juanbono) [inline src/vm/runners/cairo_runner.rs:414]: why this method is in Runner? Looks like it only needs vm

Comment (azteca1998) [reply src/vm/runners/cairo_runner.rs:414]: It was originally within `CairoRunner`.

**Review by juanbono — APPROVED** [review]

**Review by entropidelic — APPROVED** [review]

**Pull request merged**